shell/browser/api/electron_api_menu_mac.mm

```diff
@@ -127,6 +127,42 @@
                                                    std::move(close_popup));
 }
 
+std::u16string MenuMac::GetAcceleratorTextAt(int index) const {
```

Comment:
Would it make sense to name this `GetAcceleratorTextForTestAt`? I'm just trying to think of a way to surface that this is specifically meant for testing.

Reply:
Yeah it totally makes sense, and we can wrap the method in a `#ifdef DCHECK_IS_ON`, there are a few places using this pattern.